jerry-core/parser/js/js-lexer.h

```diff
@@ -270,6 +270,12 @@ typedef enum
   LEXER_STRING_RAW = (1u << 1),           /**< raw string ECMAScript v6, 11.8.6.1: TVR */
 } lexer_string_options_t;
 
+typedef enum
+{
+  LEXER_LIT_LOCATION_NO_OPTS = 0,
+  LEXER_LIT_LOCATION_HAS_ESCAPE = (1u << 0),
+  LEXER_LIT_LOCATION_HAS_UNDERSCORE = (1u << 1),
```

Comment:
Has escape and has underscore seems mutually exclusive. Btw, why do we exactly need this flag?